src/current/cockroachcloud/create-an-advanced-cluster.md

```diff
@@ -0,0 +1,186 @@
+    - For multi-region deployments, we require a minimum of 3 nodes per region. For best performance and stability, you should use the same number of nodes in each region.
+    - See [Plan a CockroachDB Cloud cluster](plan-your-cluster.html) for the requirements and recommendations for CockroachDB {{ site.data.products.advanced }} cluster configuration.
+
+        {% include cockroachcloud/nodes-limitation.md %}
+
+        Currently, you can add a maximum of 150 nodes to your cluster. For larger configurations, [contact us](https://support.cockroachlabs.com/hc/requests/new).
+
+Click **Next: Capacity**.
+
+## Step 4. Enable VPC Peering (optional)
+
+You can use [VPC peering]({% link cockroachcloud/network-authorization.md %}#vpc-peering) to connect a GCP application to a CockroachDB {{ site.data.products.cloud }} cluster deployed on GCP. A separate VPC Peering connection is required for each cluster.
+
+VPC peering is only available for GCP clusters. For clusters deployed on AWS, you can [set up AWS PrivateLink]({% link cockroachcloud/network-authorization.md %}#aws-privatelink) after creating your cluster. [Azure Virtual Network Peering](https://learn.microsoft.com/azure/virtual-network/virtual-network-peering-overview) is not yet supported. Refer to [CockroachDB {{ site.data.products.advanced }} on Azure]({% link cockroachcloud/cockroachdb-advanced-on-azure.md %}).
+
+You can use CockroachDB {{ site.data.products.cloud }}'s default IP range and size (`172.28.0.0/14`) as long as it doesn't overlap with the IP ranges in your network. Alternatively, you can configure the IP range:
+
+1. In the **VPC Peering section**, select **Configure the IP range** to configure your own IP range.
+    
+1. Enter the IP range and size (in CIDR format) for the CockroachDB {{ site.data.products.cloud }} network based on the following considerations:
+      -  As per [GCP's overlapping subnets restriction](https://cloud.google.com/vpc/docs/vpc-peering#restrictions), configure an IP range that doesn't overlap with the IP ranges in your application network.
+      - The IP range and size cannot be changed after the cluster is created. Configuring a smaller IP range size may limit your ability to expand into multiple regions in the future. We recommend configuring an IP range size of `/16` or lower.
+
+        {{site.data.alerts.callout_info}}
+        Custom IP ranges are temporarily unavailable for multi-region clusters.
+        {{site.data.alerts.end}}
+
+1. Click **Next: Capacity**.
+
+        Once your cluster is created, see [Establish VPC Peering or AWS PrivateLink]({% link cockroachcloud/connect-to-your-cluster.md %}#establish-gcp-vpc-peering-or-aws-privatelink) to finish setting up VPC Peering for your cluster.
+
+If you don't want to enable VPC Peering, leave the default selection of **Use the default IP range** as is and click **Next: Capacity**.
```

Comment:
I would swap this sentence with the previous one as an "You can do this later instead" CTA. WDYT?

Reply:
Will update content in April